libraries/AP_Camera/AP_Camera.h

```diff
@@ -127,6 +127,7 @@ class AP_Camera {
     AP_Int16        _min_interval;      // Minimum time between shots required by camera
     AP_Int16        _max_roll;          // Maximum acceptable roll angle when trigging camera
     uint32_t        _last_photo_time;   // last time a photo was taken
+    bool            _trigger_pending;
```

Comment:
maybe add a comment beside this variable's declaration to match the others..

Reply:
done